lemur/plugins/lemur_email/templates/authority_expiration.html

```diff
@@ -0,0 +1,172 @@
+                                   style="min-width:332px;max-width:600px;border:1px solid #e0e0e0;border-bottom:0;border-top-left-radius:3px;border-top-right-radius:3px">
+                                <tbody>
+                                <tr>
+                                    <td height="72px" colspan="3"></td>
+                                </tr>
+                                <tr>
+                                    <td width="32px"></td>
+                                    <td style="font-family:Roboto-Regular,Helvetica,Arial,sans-serif;font-size:24px;color:#ffffff;line-height:1.25">
+                                       Your CA certificate(s) are expiring in {{ message.options | interval }} days!
+                                    </td>
+                                    <td width="32px"></td>
+                                </tr>
+                                <tr>
+                                    <td height="18px" colspan="3"></td>
+                                </tr>
+                                </tbody>
+                            </table>
+                        </td>
+                    </tr>
+                    <tr>
+                        <td>
+                            <table bgcolor="#FAFAFA" width="100%" border="0" cellspacing="0" cellpadding="0"
+                                   style="min-width:332px;max-width:600px;border:1px solid #f0f0f0;border-bottom:1px solid #c0c0c0;border-top:0;border-bottom-left-radius:3px;border-bottom-right-radius:3px">
+                                <tbody>
+                                <tr height="16px">
+                                    <td width="32px" rowspan="3"></td>
+                                    <td></td>
+                                    <td width="32px" rowspan="3"></td>
+                                </tr>
+                                <tr>
+                                    <td>
+                                        <table style="min-width:300px" border="0" cellspacing="0" cellpadding="0">
+                                            <tbody>
+                                            <tr>
+                                                <td style="font-family:Roboto-Regular,Helvetica,Arial,sans-serif;font-size:13px;color:#202020;line-height:1.5">
+                                                    Hi,
+                                                </td>
+                                            </tr>
+                                            <tr>
+                                                <td style="font-family:Roboto-Regular,Helvetica,Arial,sans-serif;font-size:13px;color:#202020;line-height:1.5">
+                                                    <br>This is a Lemur CA certificate expiration notice. The following CA certificates are expiring soon;
+                                                    please take manual action to renew them if necessary. You may also disable notifications via the
```

Comment:
shall we add something along the lines, to ensure we point out this not a simple rotate the cert operation.

```
Note that rotating a root CA requires advanced planing and the respective trustStores need to be updated.
A subCA on the other hand, does not require any changes to the trustStore.
```

Reply:
Good idea. Ideally we could also indicate whether the CA is a root of sub-CA in the email, but I don't see any handy DB columns that could give that info, and it seems a little excessive to parse the cert to check if it's self-signed.

Reply:
ideally the number of CA certs should be low enough such that checking for self-signed would not consume too much resources.
if you would consider it as a potential improvement in the messaging, this might be what you need

https://github.com/Netflix/lemur/blob/028f0c333042346a6c155ea7692d0a5b74a27ee1/lemur/common/utils.py#L265-L275

Reply:
Also another improvement that comes to mind (maybe as part of a future PR), is to list the number of valid leaf certs under that CA, similar to the number of endpoints for the expiry notifications.
this might be a good indication, in terms of how actively this CA is being used.